Makefile

```diff
@@ -71,10 +71,10 @@ db-build: trivy-db
 
 .PHONY: db-compact
 db-compact: $(GOBIN)/bbolt cache/db/trivy.db
-	mkdir assets/
+	mkdir -p assets/
 	$(GOBIN)/bbolt compact -o ./assets/trivy.db cache/db/trivy.db
 	cp cache/db/metadata.json ./assets/metadata.json
-	rm cache/db/trivy.db
+	rm cache/db/trivy.db cache/db/metadata.json
```

Comment:
can all files be deleted?

Reply:
Yes, all the necessary files are copied under the `assets` dir at line 75-76. We can remove them under  the `cache` dir.

Reply:
Fixed https://github.com/aquasecurity/trivy-db/pull/169/commits/2776cefbbf948f7375144f0e367c042aedc076b9